### PR TITLE
Pronto message

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: lib/gooddata.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ module GoodData
 end
 
 # Modules
-require_relative 'gooddata/bricks/bricks'
+ require_relative 'gooddata/bricks/bricks'
```
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47373054581200 {line_origin: :addition, content: " require_relative 'gooddata/bricks/bricks'\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47373043624720>, repo=#<Pronto::Git::Repository:0x0000562bcadfd968 @repo=#<Rugged::Repository:47373043625120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47373054581500 {header: "@@ -12,7 +12,7 @@ module GoodData\n", count: 8}>> warning: Style/IndentationConsistency: Inconsistent indentation detected. (rubocop)

File: lib/gooddata/client.rb [hunk 2]
```diff
@@ -58,7 +58,10 @@ def init_module
 
     # Returns information about the GoodData API as a Hash (e.g. version, release time etc.)
     def release_info
-      @release_info ||= @connection.get(RELEASE_INFO_PATH)['release']
+      @release_info ||= @connection.get(RELEASE_INFO_PATH)['release']
```
Comment (yenkins) [inline lib/gooddata/client.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47373053965940 {line_origin: :addition, content: "      @release_info ||= @connection.get(RELEASE_INFO_PATH)['release'] \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47373043624680>, repo=#<Pronto::Git::Repository:0x0000562bcadfd968 @repo=#<Rugged::Repository:47373043625120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47373053966240 {header: "@@ -58,7 +58,10 @@ module GoodData\n", count: 11}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

File: lib/gooddata/client.rb [hunk 3]
```diff
@@ -58,7 +58,10 @@ def init_module
 
     # Returns information about the GoodData API as a Hash (e.g. version, release time etc.)
     def release_info
-      @release_info ||= @connection.get(RELEASE_INFO_PATH)['release']
+      @release_info ||= @connection.get(RELEASE_INFO_PATH)['release'] 
+
+
```
Comment (yenkins) [inline lib/gooddata/client.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47373053965860 {line_origin: :addition, content: "\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47373043624680>, repo=#<Pronto::Git::Repository:0x0000562bcadfd968 @repo=#<Rugged::Repository:47373043625120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47373053966240 {header: "@@ -58,7 +58,10 @@ module GoodData\n", count: 11}>> warning: Style/EmptyLines: Extra blank line detected. (rubocop)

File: lib/gooddata/client.rb [hunk 4]
```diff
@@ -58,7 +58,10 @@ def init_module
...
+
```
Comment (yenkins) [inline lib/gooddata/client.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47373053965820 {line_origin: :addition, content: "\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47373043624680>, repo=#<Pronto::Git::Repository:0x0000562bcadfd968 @repo=#<Rugged::Repository:47373043625120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47373053966240 {header: "@@ -58,7 +58,10 @@ module GoodData\n", count: 11}>> warning: Style/EmptyLines: Extra blank line detected. (rubocop)

Comment (yenkins) [inline lib/gooddata/client.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47373053965820 {line_origin: :addition, content: "\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47373043624680>, repo=#<Pronto::Git::Repository:0x0000562bcadfd968 @repo=#<Rugged::Repository:47373043625120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47373053966240 {header: "@@ -58,7 +58,10 @@ module GoodData\n", count: 11}>> warning: Style/EmptyLinesAroundMethodBody: Extra empty line detected at method body end. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/460/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1291/ : FAILURE in 1m 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1561/ : SUCCESS in 1m 10s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/461/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1292/ : FAILURE in 1m 11s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1562/ : SUCCESS in 1m 07s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/462/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1293/ : FAILURE in 58s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1563/ : SUCCESS in 1m 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/463/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1294/ : FAILURE in 1m 04s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1564/ : SUCCESS in 1m 34s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/466/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1297/ : FAILURE in 1m 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1567/ : SUCCESS in 1m 08s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/467/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1298/ : FAILURE in 1m 18s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1568/ : SUCCESS in 1m 29s

File: lib/gooddata.rb [hunk 5]
```diff
@@ -8,8 +8,8 @@
 $pmap_default_thread_count = 20 # rubocop:disable GlobalVars
 
 # GoodData Module
-module GoodData
-end
+ module GoodData
```
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47139890207560 {line_origin: :addition, content: " module GoodData\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47139881659720>, repo=#<Pronto::Git::Repository:0x000055bf37cd5dd8 @repo=#<Rugged::Repository:47139881660120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47139890208400 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Style/IndentationConsistency: Inconsistent indentation detected. (rubocop)

File: lib/gooddata.rb [hunk 6]
```diff
@@ -8,8 +8,8 @@
...
+end
```
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47139890207460 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47139881659720>, repo=#<Pronto::Git::Repository:0x000055bf37cd5dd8 @repo=#<Rugged::Repository:47139881660120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47139890208400 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Lint/EndAlignment: `end` at 12, 0 is not aligned with `module` at 11, 1. (rubocop)

Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47139890207460 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47139881659720>, repo=#<Pronto::Git::Repository:0x000055bf37cd5dd8 @repo=#<Rugged::Repository:47139881660120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47139890208400 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/468/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1299/ : FAILURE in 1m 03s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1569/ : SUCCESS in 1m 03s

File: lib/gooddata.rb (hunk 5 above)
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47112739888800 {line_origin: :addition, content: " module GoodData\n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47112719978480>, repo=#<Pronto::Git::Repository:0x000055b291e0a440 @repo=#<Rugged::Repository:47112719979020 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47112739962920 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Style/IndentationConsistency: Inconsistent indentation detected. (rubocop)

File: lib/gooddata.rb (hunk 6 above)
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47112739888760 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47112719978480>, repo=#<Pronto::Git::Repository:0x000055b291e0a440 @repo=#<Rugged::Repository:47112719979020 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47112739962920 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Lint/EndAlignment: `end` at 12, 0 is not aligned with `module` at 11, 1. (rubocop)

Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47112739888760 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47112719978480>, repo=#<Pronto::Git::Repository:0x000055b291e0a440 @repo=#<Rugged::Repository:47112719979020 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47112739962920 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/473/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1304/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1574/ : SUCCESS in 1m 04s

File: lib/gooddata.rb [hunk 7]
```diff
@@ -9,7 +9,7 @@
 
 # GoodData Module
 module GoodData
-end
+end
```
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47400965494220 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47400945722340>, repo=#<Pronto::Git::Repository:0x00005638c9104de0 @repo=#<Rugged::Repository:47400945723100 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47400965494520 {header: "@@ -9,7 +9,7 @@ $pmap_default_thread_count = 20 # rubocop:disable GlobalVars\n", count: 8}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/474/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1305/ : FAILURE in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1575/ : SUCCESS in 1m 03s

File: lib/gooddata.rb (hunk 7 above)
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47227437258180 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47227417808320>, repo=#<Pronto::Git::Repository:0x000055e7faecb3b8 @repo=#<Rugged::Repository:47227417811280 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47227437258480 {header: "@@ -9,7 +9,7 @@ $pmap_default_thread_count = 20 # rubocop:disable GlobalVars\n", count: 8}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/476/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1307/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1577/ : SUCCESS in 1m 12s

File: lib/gooddata/connection.rb [hunk 8]
```diff
@@ -4,7 +4,7 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-require 'uri'
+require 'uri'
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47310544982080 {line_origin: :addition, content: "require 'uri' \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47310533365100>, repo=#<Pronto::Git::Repository:0x0000560eaf121f88 @repo=#<Rugged::Repository:47310533365660 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47310544982380 {header: "@@ -4,7 +4,7 @@\n", count: 8}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/477/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1308/ : FAILURE in 2m 30s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1578/ : SUCCESS in 2m 30s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/479/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1310/ : FAILURE in 1m 03s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1580/ : SUCCESS in 1m 01s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/480/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1311/ : FAILURE in 1m 04s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1581/ : SUCCESS in 1m 05s

**Pull request closed without merging**